src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const filteredStyles = styles.filter(
+    (styleDeclaration) => styleDeclaration.trim() !== '',
+  );
+
+  const styleObject = {};
```

Comment:
The variable name 'styleObject' is descriptive and makes the code readable, which is good practice.